app/templates/application.hbs

```diff
@@ -4,6 +4,8 @@
   {{/es-navbar}}
 {{/es-header}}
 
+{{job-board-banner}}
```

Comment:
This goes in the index and not the application 👍 it only needs to show up on the **home** page